docs/docsite/rst/user_guide/playbooks_reuse_includes.rst

```diff
@@ -67,6 +67,8 @@ See :ref:`ansible_variable_precedence` for more details on variable inheritance
 
 Task include and import statements can be used at arbitrary depth.
 
+See :ref:`playbooks_reuse_include_tasks_example` for a worked example of include_tasks.
```

Comment:
For this link to work, you need to add an anchor to the target of the `:ref:` link - see https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#internal-navigation for more on anchors and links.

Reply:
That's what the `undefined label` error is telling you.

Reply:
Thank you. I'll make some corrections.